src/openforms/registrations/contrib/stuf_zds/plugin.py

```diff
@@ -110,6 +119,9 @@ class StufZDSRegistration(BasePlugin):
         # "initiator.aanschrijfwijze": FieldConf(RegistrationAttribute.initiator_aanschrijfwijze),
         "initiator.bsn": FieldConf(submission_field="bsn"),
         "initiator.kvk": FieldConf(submission_field="kvk"),
+        "lokatie": FieldConf(
```

Comment:
is it called `lokatie`? Shouldn't that be `locatie`?

Reply:
It is `lokatie` in Stuf but I updated it anyway.